conformance/src/main/scala/za/co/absa/enceladus/conformance/interpreter/rules/MappingRuleInterpreter.scala

```diff
@@ -126,6 +117,30 @@ case class MappingRuleInterpreter(rule: MappingConformanceRule, conformance: Con
     res2
   }
 
+  private def getDefaultValue(mappingTableDef: MappingTable)
+                             (implicit spark: SparkSession, dao: EnceladusDAO): Option[String] = {
+    val defaultMappingValueMap = mappingTableDef.getDefaultMappingValues
+
+    val attributeDefaultValueOpt = defaultMappingValueMap.get(rule.targetAttribute)
+    val genericDefaultValueOpt = defaultMappingValueMap.get("*")
+
+    val defaultValueOpt = attributeDefaultValueOpt match {
+      case Some(_) => attributeDefaultValueOpt
+      case None => genericDefaultValueOpt
+    }
+
+    if (defaultValueOpt.nonEmpty) {
```

Comment:
It's also odd that you are using `nonEmpty` (a `TraversableOnce` function added to `Option` for API interoperability, I guess, but a little confusing when actually applied to such) on an `Option`, which is an alias for `isDefined` (an `Option`-specific function).